ts/src/pro/binance.ts

```diff
@@ -1561,6 +1566,167 @@ export default class binance extends binanceRest {
         this.handleOrder (client, message);
     }
 
+    async watchPositions (symbols: string[] = undefined, since: Int = undefined, limit: Int = undefined, params = {}) {
+        /**
+         * @method
+         * @name binance#watchPositions
+         * @description watch all open positions
+         * @param {[string]|undefined} symbols list of unified market symbols
+         * @param {object} params extra parameters specific to the binance api endpoint
+         * @returns {[object]} a list of [position structure]{@link https://docs.ccxt.com/en/latest/manual.html#position-structure}
+         */
+        await this.loadMarkets ();
```

Comment:
although I have an opened position I don't get anything back, is that expected?

```
} 
ResponseBody:
 {"listenKey":"siBqU5Jfjll3G3TKk724m2G1uxAP6R0mhxe40JNqFY8gvOTnwLgwv6sjxhhZlzPl"} 

2023-05-24T15:36:11.948Z connecting to wss://stream.binancefuture.com/ws/siBqU5Jfjll3G3TKk724m2G1uxAP6R0mhxe40JNqFY8gvOTnwLgwv6sjxhhZlzPl
2023-05-24T15:36:13.213Z onUpgrade
2023-05-24T15:36:13.214Z onOpen
```
![image](https://github.com/ccxt/ccxt/assets/43336371/79a2bd22-c3fc-4a27-b6e1-e91ba8e7cf43)

Reply:
The snapshot works in the same way as watchBalance, and even though it caches it, it does not return until there is a new update. To get an upate you need to add, remove or open a new position.

I made some small changes can you try again?
Also should I go implementing in the same way python and php?

Reply:
I would say this is a bit different and we should probably return immediately if the user has any position open. 

> Also should I go implementing in the same way python and PHP?

Not yet 🙂

Reply:
@pcriadoperez I think we have a problem here,  I had a previously opened position and upon changing the position (bought 0.1 BC more) I got only that "update" not the entire position
![image](https://github.com/ccxt/ccxt/assets/43336371/dbf3f3d0-f30c-49f6-9c87-9761762fe5f7)
![image](https://github.com/ccxt/ccxt/assets/43336371/35489506-2994-4bb5-a622-ef8ea4ad8531)